Content.Shared/Atmos/Components/GasMinerComponent.cs

```diff
@@ -0,0 +1,47 @@
+using Content.Shared.Atmos;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Atmos.Components;
+
+[NetworkedComponent]
+[AutoGenerateComponentState]
+[RegisterComponent]
+public sealed partial class GasMinerComponent : Component
+{
+    [AutoNetworkedField]
+    [ViewVariables(VVAccess.ReadOnly)]
+    public bool Enabled { get; set; } = true;
+
+    [AutoNetworkedField]
+    [ViewVariables(VVAccess.ReadOnly)]
+    public bool Idle { get; set; } = false;
```

Comment:
Isn't it always idle when not enabled? This seems like it should be a 3 state enum instead of 2 bools.
Also xmldoc.